cardano-cli/src/Cardano/CLI/EraBased/Options/Governance/DRep.hs

```diff
@@ -111,15 +111,18 @@ pRegistrationCertificateCmd era = do
       GovernanceDRepRegistrationCertificateCmdArgs w
         <$> pDRepHashSource
         <*> pKeyRegistDeposit
-        <*> pDRepMetadata
+        <*> optional
+          ( pPotentiallyCheckedAnchorData
+              (pMustCheckHash drepMetadataHashCheckInfo)
+              pDRepMetadata
+          )
         <*> pOutputFile
 
-pDRepMetadata :: Parser (Maybe (L.Anchor L.StandardCrypto))
```

Comment:
Nice change :+1: It's better to have the caller do the call to `optional` indeed

Reply:
Thanks 🙂